test/test_tokeniser.py

```diff
@@ -227,3 +227,7 @@ def testAssociation():
         Token("Symbol", "m", 7),
         Token("RawRightAssociation", "|>", 8),
     ]
+
+def testIsSymbol():
```

Comment:
In Python method names, the convention is to use underscores rather than camelcase.  So `test_is_symbol`. 

Also, please adjust your editor (I thought I had adjusted `.editorconfig` to add newlines at the ends of the file.

Reply:
@rocky I just followed how the other tests [are](https://github.com/Mathics3/mathics-scanner/blob/master/test/test_tokeniser.py#L74)

Reply:
Sigh. Ok - thanks for the information. 

I guess then for now we'll live with this (like so many other things). 

I wonder what's up with black? I'll see if I can figure that out later.

Please, then just add the trailing newline, then. Thanks.

Reply:
Oh - and forgot to mention. Thanks for finding and fixing!

I need to try out with the rest of the codebase later.

 (We get s swamped in these side issues that the main issue often gets lost).

Reply:
@rocky I'll do that.